qemu/tests/cfg/qemu_guest_agent.cfg

```diff
@@ -16,6 +16,9 @@
     backup_file = /etc/sysconfig/qemu-ga-bk
     black_list_backup = /bin/cp -f /etc/sysconfig/qemu-ga ${backup_file}
     recovery_black_list = mv -f ${backup_file} /etc/sysconfig/qemu-ga
+    cmd_create_log_dict = "mkdir /var/log/journal"
+    cmd_chgrp_chmod = "chgrp systemd-journal /var/log/journal && chmod g+s /var/log/journal"
+    cmd_restart_journald = "systemctl restart systemd-journald"
```

Comment:
Are the above params shared by other test cases? Otherwise I suppose to define it only under `check_powerdown` would be more appropriate.